site/docs/4.1/utilities/borders.md

```diff
@@ -61,6 +61,7 @@ Add classes to an element to easily round its corners.
   <img data-src="holder.js/75x75" class="rounded-bottom" alt="Example bottom rounded image">
   <img data-src="holder.js/75x75" class="rounded-left" alt="Example left rounded image">
   <img data-src="holder.js/75x75" class="rounded-circle" alt="Completely round image">
+  <img data-src="holder.js/75x75" class="rounded-pill" alt="Example rounded pill image">
```

Comment:
This example is not clear, use a landscape or portrait image to indicate what the effect is.

Reply:
Could probably remove it completely from here as it's only the `.btn-rounded` that is needed. I simply added a `.rounded-pill` as per MDO's suggestion of a utility class and wasn't sure where to put it.

Reply:
No, not really, the `.rounded-pill` can still be used to add the pill effect on landscape or portrait images: https://codepen.io/MartijnCuppens/pen/QZwaJv

Reply:
No, not really, the .rounded-pill can still be used to add the pill effect on landscape or portrait images: https://codepen.io/MartijnCuppens/pen/QZwaJv